client/js/map.js

```diff
@@ -39,7 +41,10 @@ var initMarkers = function (viewportBounds, zoomLevel) {
         queryUrl = queryUrl + "&organization=" + organization;
     } else if (treeid != null) {
         queryUrl = queryUrl + "&treeid=" + treeid;
-    }
+    } else if(token == null)
+		{
+			console.log("hello world");
```

Comment:
I think here it's apparent you are using tabs instead of spaces, but the rest of this file does not. Also you should delete this